### ISSUE TITLE
Uncaught TypeError: Cannot read property 'some' of undefined

### ISSUE THREAD

**Issue by 734MNOOB** [chfritz/atom-sane-indentation#1]
[Enter steps to reproduce below:]
1. Opened up Atom with an empty doc
2. Created New File
3. Pasted in the following code (it's an regex for reddit)

```
~title (regex, full-exact): >-
[a-zA-Z0-9 \°\”\“\™\®\²\³\^\´\`\§\!\,\.\–\~\\\|\@\#\$\€\£\%\^\&\*\(\)\_\\+\-\=\{\}\;\'\:\"\/\<\>?\[\]]+
action: spam
```
1. I wanted to indent/format the code a little nicer, so I clicked on the second line (starting at "[a-z" and pressed tab
2. error is shown

**Atom Version**: 1.6.0
**System**: Mac OS X 10.11.4
**Thrown From**: [sane-indentation](https://github.com/chfritz/atom-sane-indentation) package, v0.0.5
### Stack Trace

Uncaught TypeError: Cannot read property 'some' of undefined

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:816

TypeError: Cannot read property 'some' of undefined
    at LanguageMode.suggestedIndentForTokenizedLineAtBufferRow (/Users/teamNOOB/.atom/packages/sane-indentation/lib/sane-indentation.js:106:23)
    at LanguageMode.module.exports.LanguageMode.suggestedIndentForBufferRow (/Applications/Atom.app/Contents/Resources/app.asar/src/language-mode.js:288:19)
    at TextEditor.module.exports.TextEditor.suggestedIndentForBufferRow (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:3091:32)
    at Selection.module.exports.Selection.indent (/Applications/Atom.app/Contents/Resources/app.asar/src/selection.js:798:37)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2382:26
    at /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:843:29
    at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:811:18)
    at TextEditor.module.exports.TextEditor.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1243:26)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:837:24
    at TextEditor.module.exports.TextEditor.mergeSelections (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2123:18)
    at TextEditor.module.exports.TextEditor.mergeIntersectingSelections (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2085:35)
    at TextEditor.module.exports.TextEditor.mutateSelectedText (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:835:19)
    at TextEditor.module.exports.TextEditor.indent (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2381:19)
    at TextEditor.commandRegistry.add.stopEventPropagationAndGroupUndo.editor:indent (/Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:574:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:640:34
    at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:811:18)
    at TextEditor.module.exports.TextEditor.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1243:26)
    at atom-text-editor.newCommandListeners.(anonymous function) (/Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:639:22)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:570:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:378:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```
### Commands

```
     -0:18.9.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:18.9.0 editor:indent (atom-text-editor.editor.is-focused)
     -0:18.8.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:18.8.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:18.8.0 editor:indent (atom-text-editor.editor.is-focused)
     -0:18.6.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:18.6.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:18.6.0 editor:indent (atom-text-editor.editor.is-focused)
     -0:18.4.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:18.4.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:18.4.0 editor:indent (atom-text-editor.editor.is-focused)
  3x -0:10.9.0 core:close (atom-text-editor.editor.is-focused)
     -0:03.2.0 core:paste (atom-text-editor.editor.is-focused)
     -0:01.6.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:01.6.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:01.6.0 editor:indent (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "audioBeep": false,
    "disabledPackages": [
      "css-clean"
    ],
    "projectHome": "/Users/teamNOOB/Sites"
  }
}
```
### Installed Packages

``` coffee
# User
cssnano, v1.1.0
preview-plus, v1.2.6
sane-indentation, v0.0.5
you-complete-me, v0.7.4

# Dev
No dev packages
```

**Comment by gabmontes** [comment]
Same issue. Atom is not usable at all, guys!

Had to remove the package.

**Comment by chfritz** [comment]
Could you check whether you are getting the same error with the latest version of the package? (0.0.6)

**Comment by 734MNOOB** [comment]
Same issue: 0.0.6

**"/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:816"**

`TypeError: Cannot read property 'some' of undefined
    at LanguageMode.suggestedIndentForTokenizedLineAtBufferRow (/Users/teamNOOB/.atom/packages/sane-indentation/lib/sane-indentation.js:106:23)
    at LanguageMode.module.exports.LanguageMode.suggestedIndentForBufferRow (/Applications/Atom.app/Contents/Resources/app.asar/src/language-mode.js:288:19)
    at TextEditor.module.exports.TextEditor.suggestedIndentForBufferRow (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:3091:32)
    at Selection.module.exports.Selection.indent (/Applications/Atom.app/Contents/Resources/app.asar/src/selection.js:798:37)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2382:26
    at /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:843:29
    at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:811:18)
    at TextEditor.module.exports.TextEditor.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1243:26)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:837:24
    at TextEditor.module.exports.TextEditor.mergeSelections (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2123:18)
    at TextEditor.module.exports.TextEditor.mergeIntersectingSelections (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2085:35)
    at TextEditor.module.exports.TextEditor.mutateSelectedText (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:835:19)
    at TextEditor.module.exports.TextEditor.indent (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:2381:19)
    at TextEditor.commandRegistry.add.stopEventPropagationAndGroupUndo.editor:indent (/Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:574:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:640:34
    at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:811:18)
    at TextEditor.module.exports.TextEditor.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1243:26)
    at atom-text-editor.newCommandListeners.(anonymous function) (/Applications/Atom.app/Contents/Resources/app.asar/src/register-default-commands.js:639:22)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)`

**Comment by chfritz** [comment]
Thanks for checking!

I just published another version (0.0.7), which should address the issue you are seeing. Please verify and close the issue if positive.

**Comment by 734MNOOB** [comment]
No problems! It works! Congrats :)
